### DISCUSSION TITLE
"Application cannot be reviewed"

### DISCUSSION THREAD

**Discussion by mfnalex** [community/community#42205] [GitHub Education]
Hi, when I try to upload my enrollment letter and apply for the student package, I only get the following message:

> Your application cannot be reviewed until you fix the following:
Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?,You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.

![image](https://user-images.githubusercontent.com/1122571/208680642-991feab0-dcfc-44b4-9c23-3464b811892c.png)

However, I fulfill all those requirements. What can I do to actually get my application reviewed by a human?

**Comment by mfnalex** [comment]
Any updates on this? The official GitHub support sent me here, so I expected to get an answer here.

**Reply by Kataneco** [reply]
All I can say is I have the same issue :P

**Comment by jfullstackdev** [comment]
that's correct, posting here can accelerate to solve this problem when Staff members saw this and responded, however, it's not always like that and there is no assurance

**Comment by Mohaaav** [comment]
I have the same issue ?!

**Reply by Kataneco** [reply]
so true
